.svglintrc.js

```diff
@@ -1,7 +1,10 @@
 const data = require("./_data/simple-icons.json");
 const { htmlFriendlyToTitle } = require("./scripts/utils.js");
+const getBounds = require('svg-path-bounding-box');
```

Comment:
Just for consistency, since a few changes are required anyways: 

```suggestion
const getBounds = require("svg-path-bounding-box");
```